test/common/testvm.py

```diff
@@ -1032,8 +1031,26 @@ def qemu_console(self):
                 # the domain may have already been freed (shutdown) while the console was running
                 self.message("libvirt error during shutdown: %s" % (le.get_error_message()))
 
-        except:
-            raise
+        except OSError, ex:
+            raise Failure("Failed to launch virsh command: {0}".format(ex.strerror))
+        finally:
+            self._cleanup()
+
+    def graphics_console(self):
+        self.message("Started machine {0}".format(self.label))
+        if self.maintain:
+            message = "\nWARNING: Uncontrolled shutdown can lead to a corrupted image\n"
+        else:
+            message = "\nWARNING: All changes are discarded, the image file won't be changed\n"
+        message += "\nIn the machine a web browser can access Cockpit on host:\n\nhttps://10.111.112.1:9090"
```

Comment:
Is this true? On mine that is cockpit running on my local system. I don't have the windows image to try yet, so i'm not sure.